bridges/NordbayernBridge.php

```diff
@@ -33,8 +33,7 @@ class NordbayernBridge extends BridgeAbstract {
 				'Neustadt/Aisch' => 'neustadt-aisch',
 				'Pegnitz' => 'pegnitz',
 				'Roth' => 'roth',
-				'Rothenburg o.d.T.' => 'rothenburg-o-d-t',
-				'Schwabach' => 'schwabach',
+				'Rothenburg o.d.T.' => 'rothenburg-ob-der-tauber',
```

Comment:
Old feed urls with these params will become invalid if following change is accepted. Can we escape it?

Reply:
done